routers/cart-router.js

```diff
@@ -0,0 +1,93 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const {Cart} = require('../models/cart-model')
+const CartRouter   = express.Router()
+CartRouter.use(bodyParser.json({limit: '5000kb'}))
+const cors = require('cors')
+CartRouter.use(cors())
+CartRouter.route('/')
+.get(async (req , res) =>{
+  try{
+     const myCart = await Cart.find({})
+     res.json({myCart :myCart})
+  }catch(error){
+    res.json({status : false  , message : "couldn't collect the data" , errorMessaage : error.message})
+  }
+ 
+})
+.post(async (req , res ) =>{
+  try{
+    const data = req.body
+    const newProduct =new  Cart(data)
+    const SavedData = await newProduct.save()
+    res.json({SavedData : SavedData})
+  }
+  catch(error){
+    res.json({status : false , message : "failed to get the cart" , errorMessaage : error.message})
+  }
+})
+
+.put(async (req , res) =>{
+  const id = req.body._id
+const paramCase = req.param('case')
+  // for increment
+  if(paramCase === 'inc'){
+     try{
+      Cart.updateOne(
+    {_id :id},
+     {
+       $inc : {quantity : 1 }
+       } , 
+       
+    (err,result) =>  {
+      if (err) {
+        res.send(err);
+      } else {
+        res.json(result);
+      }
+     }
+   )
+  }
+catch(error){
+  res.json({message : "failed" , errMessage :error.message})
+}
+  }  //if close
+
+  //for decrement
+  else{
+       try{
+      Cart.updateOne(
+    {_id :id},
+     {
+       $inc : {quantity : -1 }
+       } , 
+       
+    (err,result) =>  {
+      if (err) {
+        res.send(err);
+      } else {
+        res.json(result);
+      }
+     }
+   )
+  }
+catch(error){
+  res.json({message : "failed" , errMessage :error.message})
+}
+  }
```

Comment:
What you can do here instead is extract the increment and decrement operation to their own separate routes. Then it would be easier to make out what each route is doing.